hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/metrics2/lib/MutableQuantiles.java

```diff
@@ -104,8 +105,7 @@ public MutableQuantiles(String name, String description, String sampleName,
           String.format(descTemplate, percentile));
     }
 
-    estimator = new SampleQuantiles(quantiles);
-
+    estimator = inverseQuantiles ? new InverseQuantiles(quantiles) : new SampleQuantiles(quantiles);
```

Comment:
Is estimator variable used?

Reply:
yes, estimator is the one which will sort the sample and populate quantiles.